pkg/onboarding/usecases/facility/facility_integration_test.go

```diff
@@ -0,0 +1,77 @@
+package facility_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/savannahghi/onboarding-service/pkg/onboarding/application/dto"
+)
+
+func TestUseCaseFacilityImpl_DeleteFacility_Integration_Test(t *testing.T) {
```

Comment:
This test should be _table driven_
_Create a **Facility** _
Confirm the Facility exists
Delete the facility 
_CONFIRM, CONFIRM and CONFIRM_